File: tools/output/cpp/performance.cpp
#include "performance.h"

#include <string>
/* #include_block */
using std::string;
using mei::MeiAttribute;
using mei::MeiNamespace;
using mei::AttributeNotFoundException;

mei::AvFile::AvFile() :
    MeiElement("avFile")
{
}
REGISTER_DEFINITION(mei::AvFile, "avFile");
mei::AvFile::~AvFile() {}
mei::AvFile::AvFile(const AvFile& other) :
    MeiElement(other),

{
}

/* include <avFile> */

mei::Clip::Clip() :
    MeiElement("clip")
{
}
REGISTER_DEFINITION(mei::Clip, "clip");
mei::Clip::~Clip() {}
mei::Clip::Clip(const Clip& other) :
    MeiElement(other),

{
}

/* include <clip> */

mei::Performance::Performance() :
    MeiElement("performance")
{
}
REGISTER_DEFINITION(mei::Performance, "performance");
mei::Performance::~Performance() {}
mei::Performance::Performance(const Performance& other) :
    MeiElement(other),

{
}

/* include <performance> */

mei::Recording::Recording() :
    MeiElement("recording")
{
}
REGISTER_DEFINITION(mei::Recording, "recording");
mei::Recording::~Recording() {}
mei::Recording::Recording(const Recording& other) :
    MeiElement(other),

{
}

/* include <recording> */

File: python/pymei/Modules/performance.py

"""
    Copyright (c) 2011-2013 Andrew Hankinson, Alastair Porter, and Others
    
    Permission is hereby granted, free of charge, to any person obtaining
    a copy of this software and associated documentation files (the
    "Software"), to deal in the Software without restriction, including
    without limitation the rights to use, copy, modify, merge, publish,
    distribute, sublicense, and/or sell copies of the Software, and to
    permit persons to whom the Software is furnished to do so, subject to
    the following conditions:
    
    The above copyright notice and this permission notice shall be
    included in all copies or substantial portions of the Software.
    
    THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND,
    EXPRESS OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF
    MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND
    NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT HOLDERS BE
    LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION
    OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION
    WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
"""

from pymei import MeiElement


class avFile_(MeiElement):
    def __init__(self):
        MeiElement.__init__(self, "avFile")
    # <avFile>

class clip_(MeiElement):
    def __init__(self):
        MeiElement.__init__(self, "clip")
    # <clip>

class performance_(MeiElement):
    def __init__(self):
        MeiElement.__init__(self, "performance")
    # <performance>

class recording_(MeiElement):
    def __init__(self):
        MeiElement.__init__(self, "recording")
    # <recording>

class when_(MeiElement):
    def __init__(self):
        MeiElement.__init__(self, "when")
    # <when>

